ui/perfherder/alerts/Assignee.jsx

```diff
@@ -90,12 +90,17 @@ export default class Assignee extends React.Component {
 
   render() {
     const { user } = this.props;
-    const { assigneeUsername, newAssigneeUsername, inEditMode } = this.state;
+    const { assigneeUsername, inEditMode } = this.state;
+    let newAssigneeUsername;
+    if (!this.state.newAssigneeUsername) {
+      newAssigneeUsername = ""
+  } else {
+      newAssigneeUsername = this.state.newAssigneeUsername
+  }
```

Comment:
First, is `newAssigneeUsername` ever falsey now that you have set it to default empty string?  If so, then you do not need this logic.  Otherwise, it is better to write:

    const newAssigneeUsername = this.state.newAssigneeUsername || "";